lib/bamboo/test.ex

```diff
@@ -157,12 +157,9 @@ defmodule Bamboo.Test do
       unsent_email = Bamboo.Email.new_email(subject: "something else")
       assert_delivered_email(unsent_email) # Will fail
   """
-  defmacro assert_delivered_email(email) do
-    quote do
-      import ExUnit.Assertions
-      email = Bamboo.Test.normalize_for_testing(unquote(email))
-      assert_receive({:delivered_email, ^email}, 100, Bamboo.Test.flunk_with_email_list(email))
-    end
+  def assert_delivered_email(email) do
+    email = normalize_for_testing(email)
+    assert_receive({:delivered_email, ^email}, 100, flunk_with_email_list(email))
```

Comment:
Do these all need to get updated with the timeout from the other PR?

Reply:
Yes, these will all have to get updated once https://github.com/thoughtbot/bamboo/pull/605 gets merged (or vice versa). I just wanted to introduce them cleanly in separate PRs, and I can resolve the conflicts once I merge one of them.

Reply:
I've rebased this now that the timeout PR has been merged, so this should now be fixed.